src/contracts/Protocol/Vault.sol

```diff
@@ -156,7 +159,9 @@ contract Vault is Ownable, OnlyCostController {
             metric.transfer(_msgSender(), toWithdraw);
 
             emit Withdraw(_msgSender(), toWithdraw);
-        } else if (stage == 1) {
+        } else if (stage == STAGE.UNVOTE) {
```

Comment:
not yet - but as this if/else tree grows we might want to refactor the functionality into their own methods

Reply:
Yep!